### PR TITLE
Fixed code block style (MD040) in the Software Update Guide

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes all MD040 errors in the Software Update Guide.

The MD040 rule requires that a language be specified for fenced code blocks. However, it also identifies indented code blocks, so you'll see changes in this PR that replaces indentation with fences.

This is one of many PRs related to #5252.

### CODE REVIEW

File: _includes/comp-man/checklist.md [hunk 1]
```diff
@@ -17,17 +17,23 @@ Magento requires three cron jobs, all running as the [Magento file system owner]
...
+```bash
+crontab -u magento_user -l
+```
 
 Results similar to the following should display:
 
-	* * * * * /usr/bin/php /var/www/magento2/bin/magento cron:run | grep -v "Ran jobs by schedule" >> /var/www/magento2/var/log/magento.cron.log
-	* * * * * /usr/bin/php /var/www/magento2/update/cron.php >> /var/www/magento2/var/log/update.cron.log
-	* * * * * /usr/bin/php /var/www/magento2/bin/magento setup:cron:run >> /var/www/magento2/var/log/setup.cron.log
+```terminal
```
Comment (shrielenee) [inline _includes/comp-man/checklist.md]: `bash` commands you enter + `terminal` output?

Comment (jeff-matthews) [reply _includes/comp-man/checklist.md]: Yes

File: guides/v2.2/comp-mgr/upgrader/upgrade.md [hunk 2]
```diff
@@ -37,15 +37,25 @@ bin/magento cache:clean
 
 After the upgrade completes, manually clear `var` subdirectories:
 
-	rm -rf <Magento install dir>/var/cache/*
-	rm -rf <Magento install dir>/var/page_cache/*
-	rm -rf <Magento install dir>/generated/code/*
+```bash
```
Comment (keharper) [inline guides/v2.2/comp-mgr/upgrader/upgrade.md]: You should be able to delete the contents of the three directories in one command.

File: guides/v2.2/comp-mgr/upgrader/upgrade.md [hunk 3]
```diff
@@ -70,7 +81,9 @@ Clear the `var/cache`, 	`var/page_cache`, `generated/code`
 
 A sample command follows:
 
-	rm -rf var/cache/* var/page_cache/* generated/code/*
+```bash
+rm -rf var/cache/* var/page_cache/* generated/code/*
```
Comment (keharper) [inline guides/v2.2/comp-mgr/upgrader/upgrade.md]: Aha! I was right.

**Pull request merged**